umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/iso19115_2.clj

```diff
@@ -21,20 +21,20 @@
    [cmr.umm-spec.util :as su :refer [char-string]]))
 
 (def iso19115-2-xml-namespaces
-  {:xmlns:xs "http://www.w3.org/2001/XMLSchema"
-   :xmlns:gmx "http://www.isotc211.org/2005/gmx"
-   :xmlns:gss "http://www.isotc211.org/2005/gss"
+  {:xmlns:eos "http://earthdata.nasa.gov/schema/eos"
    :xmlns:gco "http://www.isotc211.org/2005/gco"
-   :xmlns:xsi "http://www.w3.org/2001/XMLSchema-instance"
    :xmlns:gmd "http://www.isotc211.org/2005/gmd"
    :xmlns:gmi "http://www.isotc211.org/2005/gmi"
    :xmlns:gml "http://www.opengis.net/gml/3.2"
-   :xmlns:xlink "http://www.w3.org/1999/xlink"
-   :xmlns:eos "http://earthdata.nasa.gov/schema/eos"
-   :xmlns:srv "http://www.isotc211.org/2005/srv"
+   :xmlns:gmx "http://www.isotc211.org/2005/gmx"
+   :xmlns:gsr "http://www.isotc211.org/2005/gsr"
+   :xmlns:gss "http://www.isotc211.org/2005/gss"
    :xmlns:gts "http://www.isotc211.org/2005/gts"
-   :xmlns:swe "http://schemas.opengis.net/sweCommon/2.0/"
-   :xmlns:gsr "http://www.isotc211.org/2005/gsr"})
+   :xmlns:srv "http://www.isotc211.org/2005/srv"
+   :xmlns:xlink "http://www.w3.org/1999/xlink"
+   :xmlns:xs "http://www.w3.org/2001/XMLSchema"
+   :xmlns:xsi "http://www.w3.org/2001/XMLSchema-instance"
+   :xsi:schemaLocation "http://earthdata.nasa.gov/schema/eos https://cdn.earthdata.nasa.gov/iso/eos/1.0/eos.xsd http://www.isotc211.org/2005/gco https://cdn.earthdata.nasa.gov/iso/gco/1.0/gco.xsd http://www.isotc211.org/2005/gmd https://cdn.earthdata.nasa.gov/iso/gmd/1.0/gmd.xsd http://www.isotc211.org/2005/gmi https://cdn.earthdata.nasa.gov/iso/gmi/1.0/gmi.xsd http://www.opengis.net/gml/3.2 https://cdn.earthdata.nasa.gov/iso/gml/1.0/gml.xsd http://www.isotc211.org/2005/gmx https://cdn.earthdata.nasa.gov/iso/gmx/1.0/gmx.xsd http://www.isotc211.org/2005/gsr https://cdn.earthdata.nasa.gov/iso/gsr/1.0/gsr.xsd http://www.isotc211.org/2005/gss https://cdn.earthdata.nasa.gov/iso/gss/1.0/gss.xsd http://www.isotc211.org/2005/gts https://cdn.earthdata.nasa.gov/iso/gts/1.0/gts.xsd http://www.isotc211.org/2005/srv https://cdn.earthdata.nasa.gov/iso/srv/1.0/srv.xsd"})
```

Comment:
I have broken up the schemaLocation as you stated.  The name spaces tell a user what standard is being used and the date for the standard.  The schema location points the user to the schema location that the CMR is actually using to validate the records. This is the MENDS version that has been slightly modified for NASA's purpose.